Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -70,9 +70,11 @@ namespace AzToolsFramework
                 entityIds, inputEntityList, topLevelEntities, commonRootEntityId, commonRootEntityOwningInstance);
             if (!findCommonRootOutcome.IsSuccess())
             {
-                return findCommonRootOutcome;
+                return AZ::Failure(findCommonRootOutcome.TakeError());
```

Comment:
If the outcome is not success, we will be returning a failure only. Why do we need to create a new failure object again here?

Reply:
Because the return value's type has been changed to CreatePrefabResult. Can't return findCommonRootOutcome directly due to different types.